cmd/addvalidators.go

```diff
@@ -27,12 +29,14 @@ import (
 
 // addValidatorsConfig is config for the `add-validators` command.
 type addValidatorsConfig struct {
-	Lockfile          string
-	EnrPrivKeyfiles   []string
 	NumVals           int
 	WithdrawalAddrs   []string
 	FeeRecipientAddrs []string
 
+	Lockfile            string
+	ClusterManifestFile string
```

Comment:
seems like we will first read the `cluster-manifest.pb` file and then we will override the same after `add-validators` command. I'd suggest to write the new cluster manifest file in a different location rather than overriding the existing one on disk. For ex: having a `--data-dir` flag to write the new cluster manifest file.

Reply:
Cool, added a new flag `--cluster-backup-dir` to store the cluster manifest backups. The backups are in the format `cluster-manifest-backup-1687250009.pb`